receiver/prometheusremotewritereceiver/factory.go

```diff
@@ -22,7 +23,11 @@ func NewFactory() receiver.Factory {
 }
 
 func createDefaultConfig() component.Config {
-	return &Config{}
+	return &Config{
+		ServerConfig: confighttp.ServerConfig{
+			Endpoint: "localhost:9090",
```

Comment:
Is 9090 the normal default port for PRW (is it what Prom uses?).  The only other thing we might want to check is to see if any other components use 9090 as their default port.  It doesn't mean we can't too, but it would be good to know.

Reply:
Yes, Prometheus runs everything on the same port: 9090.

There's no automated way to check that, is there? 🥲

Reply:
I did a quick search, and it looks like the only thing this is likely to conflict with is the prometheus exporter. I think we should move forward with this as-is to match the prometheus server.